pkg/cassandra/datacenter.go

```diff
@@ -378,6 +379,12 @@ func Coalesce(clusterName string, clusterTemplate *api.CassandraClusterTemplate,
 		dcConfig.ManagementApiAuth = clusterTemplate.DatacenterOptions.ManagementApiAuth
 	}
 
+	if dcTemplate.DatacenterOptions.PerNodeInitContainerImage != nil {
```

Comment:
Beware that when #780 is going to be merged, there will be a conflict here. I can help you solve the conflict if need be.

Reply:
Thank you for your comments, I have rebased commit on current main branch